nanopub/java_wrapper.py

```diff
@@ -73,5 +73,12 @@ def _get_signed_file(unsigned_file: str):
         return str(unsigned_file.parent / f'signed.{unsigned_file.name}')
 
     @staticmethod
-    def make_keys():
-        subprocess.run([NANOPUB_SCRIPT, 'mkkeys', '-a', 'RSA'])
+    def make_keys(path_name='~/.nanopub/id'):
+        """
+        Use nanopub-java to make the RSA keys for this user.
+        By default, this uses the path name ~/.nanopub/id and produces a key-pair:
+            ~/.nanopub/id_rsa and ~/.nanopub/id_rsa.pub
+
+        NOTE THAT THE JAVA TOOL ADDS _rsa TO THE END OF YOUR PATH.
+        """
+        subprocess.run([NANOPUB_SCRIPT, 'mkkeys', '-a', 'RSA', '-f', path_name], check=True)
```

Comment:
I guess this is only necessary for testing?

Reply:
I was just wondering why this is.

Reply:
It does make the testing easier, but it's also to sanitize the tool a little bit. The comments and code all imply that the USERCONFIG directory can be something other than ```~/.nanopub/```, and allow it to be changed. However, by default the nanopub java tool always makes keys in this default directory, which is unexpected behaviour IMO. I've made it use the actual ```USERCONFIG``` dir for everything profile related now.